indexer/index.py

```diff
@@ -23,11 +23,13 @@
 # Types --------------------------------------------------------------------------------
 
 
-class RestClientArgs(TypedDict):
-    """TypedDict for RestClient parameters."""
+class ClientCredentialsAuthArgs(TypedDict):
+    """TypedDict for ClientCredentialsAuth parameters."""
 
-    url: str
-    token: str
+    address: str
+    token_url: str
+    client_id: str
+    client_secret: str
     timeout: int
     retries: int
```

Comment:
Reminds me of when I was really into `TypedDict` for decreasing # of args... :smile: No changes needed